op-e2e/actions/proofs/operator_fee_test.go

```diff
@@ -38,6 +37,8 @@ func Test_ProgramAction_OperatorFeeConstistency(gt *testing.T) {
 	// The deployed bytecode below is from the contract above
 	testStorageUpdateContractCode := common.FromHex("0x6080604052348015600e575f80fd5b50600436106026575f3560e01c806360fe47b114602a575b5f80fd5b60406004803603810190603c9190607d565b6042565b005b805f8190555050565b5f80fd5b5f819050919050565b605f81604f565b81146068575f80fd5b50565b5f813590506077816058565b92915050565b5f60208284031215608f57608e604b565b5b5f609a84828501606b565b9150509291505056fea26469706673582212201712a1e6e9c5e2ba1f8f7403f5d6e00090c6fa2b70c632beea4be8009331bd2064736f6c63430008190033")
 
+	l1InfoDepositorAddress := common.HexToAddress("0xdeaddeaddeaddeaddeaddeaddeaddeaddead0001")
```

Comment:
What's this address? I don't see it involved in any actual tx? Why should its balance change?

Reply:
Ah I see. Please use the public constant `derive.L1InfoDepositerAddress`

Reply:
I think everything related to the l1 info depositor can be deleted

Reply:
I updated to use Alice address